modules/ai/cognitiveservices/main.bicep

```diff
@@ -0,0 +1,175 @@
+@description('The kind of Cognitive Service to create. See: https://learn.microsoft.com/en-us/azure/cognitive-services/create-account-bicep for available kinds.')
+param kind string
```

Comment:
can we add 'Allowed' annotation here with the list of "kinds".
(totally optional, but perhaps a good place to create a user-defined type)

```suggestion
@allowed(['CognitiveServices', 'ComputerVision', 'CustomVision.Prediction', 'CustomVision.Training', 'Face', 'FormRecognizer', 'SpeechServices', 'LUIS', 'QnAMaker', 'TextAnalytics', 'TextTranslation', 'AnomalyDetector', 'ContentModerator', 'Personalizer', 'OpenAI'])
param kind string
```